README.md

```diff
@@ -13,6 +13,18 @@ Aptos is a layer 1 blockchain bringing a paradigm shift to Web3 through better t
 
 ## Getting Started
 
+### Quick
+
+```sh
+git clone https://github.com/aptos-labs/aptos-core.git
+cd aptos-core
+./scripts/dev_setup.sh
+cargo build
+cargo test
+```
+
+### Resources
+
 * [Aptos Labs](https://aptoslabs.com/)
```

Comment:
@davidiw should this link to aptoslabs.com come out of our README?

Reply:
I found the links ok, not too much.

Aptos Labs are the makers, so i would say a link  belongs in the readme.

Reply:
I removed it as we're working to make all of this clearer.

Reply:
> Make all of this clearer.

Aptos Labs are the makers, 

Aptos Foundation the copyright holders,